src/test/suite/helper.ts

```diff
@@ -3,42 +3,75 @@
  *  Licensed under the MIT License. See License.txt in the project root for license information.
  *--------------------------------------------------------------------------------------------*/
 
-import { workspace, TextLine, DocumentSelector, Event, NotebookCell, NotebookCellKind, NotebookCellOutput, NotebookConcatTextDocument, NotebookDocument, notebooks, Position, Range, TextDocument, Uri } from 'vscode';
-import { IVSCodeNotebook } from '../../common/types';
+import { assert } from 'chai';

```

Comment:
Most of the code I added here I just copied from the helpers in vscode-jupyter.

Reply:
Might move that to another NPM in the future.

Reply:
So you really don't need to review most of this file. Just the last couple of functions related to starting up the language server.